packages/trader/src/AppV2/Components/TradeParameters/Multiplier/multiplier.tsx

```diff
@@ -1,24 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
 import clsx from 'clsx';
 import { observer } from 'mobx-react';
-import { TextField } from '@deriv-com/quill-ui';
+import { ActionSheet, Skeleton, TextField } from '@deriv-com/quill-ui';
 import { Localize } from '@deriv/translations';
 import { useTraderStore } from 'Stores/useTraderStores';
+import Carousel from 'AppV2/Components/Carousel';
+import CarouselHeader from 'AppV2/Components/Carousel/carousel-header';
+import TradeParamDefinition from 'AppV2/Components/TradeParamDefinition';
+import MultiplierWheelPicker from './multiplier-wheel-picker';
 
 type TMultiplierProps = {
     is_minimized?: boolean;
 };
 
 const Multiplier = observer(({ is_minimized }: TMultiplierProps) => {
-    const { multiplier } = useTraderStore();
+    const { multiplier, multiplier_range_list, commission, onChange, currency } = useTraderStore();
+
+    const [isOpen, setIsOpen] = useState(false);
+    const is_small_screen_device = window.innerHeight <= 640;
+    const classname = clsx('trade-params__option', is_minimized && 'trade-params__option--minimized');
+
+    const handleMultiplierChange = (multiplier: number) => {
+        onChange({ target: { name: 'multiplier', value: multiplier } });
+    };
+
+    const action_sheet_content = [
+        {
+            id: 1,
+            component: (
+                <MultiplierWheelPicker
+                    multiplier={multiplier}
+                    multiplier_range_list={multiplier_range_list}
+                    currency={currency}
+                    commission={commission}
+                    setMultiplier={handleMultiplierChange}
+                />
+            ),
+        },
+        {
+            id: 2,
+            component: <TradeParamDefinition description={<Localize i18n_default_text='Content goes here' />} />,
+        },
+    ];
+
+    if (!multiplier)
+        return (
+            <div className={classname}>
+                <Skeleton.Square />
```

Comment:
sadly, the current `<Skeleton.Square />` needs `height` to work while `width` can be omitted

Reply:
In that case, it should throw an ts error, right?

Reply:
not anymore. Quill team updated height and width to be 100% of the container. That release version is already in my PR.

Reply:
p.s.: the error was not thrown but UI broke. :) not the case anymore with the new version)